addons/web/static/tests/views/list_tests.js

```diff
@@ -1901,7 +2023,7 @@ QUnit.module('Views', {
                         '<field name="date"/>' +
                         '<field name="datetime"/>' +
                         '<field name="amount"/>' +
-                        '<button name="the_button" width="25px"></button>' +
```

Comment:
Why was this case removed ?

Reply:
Setting a width on a button column is no longer supported. But actually, it wasn't **really** supported. In rng, a button doesn't have "width" as valid attribute, so you actually can't save a list view like this one. I worked for list in form views though, because form views are rng validated.

Since we do not have the button <-> column equivalence anymore (like we for fields, for instance), setting a width on a button doesn't make sense anymore. So we decided with @odoongr not to handle this usecase.